cmd/mycelo/genesis_commands.go

```diff
@@ -127,6 +140,33 @@ func envFromTemplate(ctx *cli.Context, workdir string) (*env.Environment, *genes
 		genesisConfig.Blockchain.BlockGasLimit = ctx.Uint64("blockgaslimit")
 	}
 
+	if ctx.IsSet("forks.churrito") {
+		churritoBlockNumber := ctx.Int64("forks.churrito")
+		if churritoBlockNumber < 0 {
+			genesisConfig.Hardforks.EspressoBlock = nil
```

Comment:
should it be `genesisConfig.Hardforks.ChurritoBlock = nil`?

Reply:
Yes! Nice catch